web/admin/src/Admin.js

```diff
@@ -409,7 +409,10 @@ export default class Admin extends Component {
         />
         <p className='boxTitleBoldMargin'>Moderation:   </p>
         <ModIcon moderator = {this.state.moderator} offApprove = {this.offApprove} onApprove = {this.onApprove} />
-        {this.state.session === "All" ? null : <span className="buttonSpanMargin">
+        {this.state.session === "All" ? <span className="buttonSpanMargin">
+          <div style={{flex: 1}}/>
+          <button className="overlay-button-opaque" disabled={true}>Launch in new tab</button>
```

Comment:
For attributes with `true` values, you can just say, e.g., `<button disabled>...`

Reply:
Spacing makes this hard to read. Consider:

```jsx
{ this.state.session === "All"
  ? <span className="buttonSpanMargin">
      <div style={{flex: 1}}/>
      <button className="overlay-button-opaque" disabled={true}>Launch in new tab</button>
    </span>
  : <span className="buttonSpanMargin">
      ...
    </span>
}
```